webapp/app/templates/lotse/form_steuernummer.html

```diff
@@ -0,0 +1,185 @@
+{% extends 'basis/form_standard.html' %}
+{% import "macros.html" as macros %}
+
+
+{% block form_content %}
+    <div class="form-row-center">
+        {{ macros.render_field(form.steuernummer_exists, first_field=True if not render_info.form.errors) }}
+    </div>
+    <div class="form-row-center">
+        {{ macros.render_field(form.bundesland) }}
+    </div>
+
+    <div class="form-row-center">
+        {{ macros.render_field(form.bufa_nr) }}
+    </div>
+
+    <div class="form-row-center">
+        {{ macros.render_field(form.steuernummer) }}
+    </div>
+
+    <div class="additional-input-content" for="request_new_tax_number">
+        <h2 class="form-sub-heading-smaller">{{ _('form.lotse.request_new_tax_number.title') }}</h2>
+        {{ _('form.lotse.request_new_tax_number.intro') }}
+        {{ macros.render_field(form.request_new_tax_number) }}
+    </div>
+
+{% endblock %}
+
+{% block optional_js %}
+    {{ super() }}
+    <script src="{{ url_for('static', filename='js/field_utils.js') }}" type=text/javascript></script>
+    <script>
+        let taxOffices = JSON.parse('{{ render_info.additional_info['tax_offices']|tojson }}');
+        let TAX_NUMBER_FORM_STATES = {
+            'NothingSelected': 'NothingSelected',
+            'StateSelectionTaxNumberExists': 'StateSelectionTaxNumberExists',
+            'StateSelectionNoTaxNumberExists': 'StateSelectionNoTaxNumberExists',
+            'TaxOfficeSelection': 'TaxOfficeSelection',
+            'TaxNumberInput': 'TaxNumberInput',
+            'RequestNewTaxNumber': 'RequestNewTaxNumber'
+        }
+        let taxNumberFormState = TAX_NUMBER_FORM_STATES.NothingSelected;
+
+        document.addEventListener('DOMContentLoaded', () => {
+            // Register event handlers
+            document.querySelectorAll('input[type=radio][name=steuernummer_exists]').forEach((field)=>{
+                field.addEventListener("click",() => {changeTaxNumberExists(); runThroughAllStates(); handleState();});
```

Comment:
Frede mentioned something. I used change in the beginning. However, using `change` does not work here because we use bootstraps `data-toggle` for the yes-no-field. As far as I can tell, bootstrap overwrites and prevents the default behavior if data-toggle is not set on the target directly, which is the case for us (s. [here](https://github.com/minhur/bootstrap-toggle/issues/174#issuecomment-614745518) and [here](https://github.com/twbs/bootstrap-sass/blob/master/assets/javascripts/bootstrap/button.js#L115)). 
In the bootstrap [documentation](https://getbootstrap.com/docs/3.4/javascript/), they use Jquery `change` which works. 
As far as I can tell, we could either:
- get rid of bootstrap at this point
- change how the YesNo-Field is put together 
- we keep the click event

Personally, I would think that we should keep this in mind, when we compentonize this with React but keep the click event until then.

Reply:
I think I disagree with you here. As Niko said, `click` won't work for users without a mouse, so e.g. screen reader users. And that they can use the site should in my opinion be more important than having a nice animation for sighted people. So I'd remove the `data-toggle` instead (or maybe even implement the animation ourselves?) You're very welcome to disagree with me again. :D

Reply:
I agree with @frederike-ramin that using `click` is out of the question.

How about just using jQuery for that one case with a descriptive comment? We'll have to go over this again anyway once we reactify it and then we can decide wholesale whether we want to drop bootstrap's JS components or use them via something like [react-bootstrap](https://react-bootstrap.github.io/) (which reimplements bootstrap's JS without jquery).

Reply:
> And that they can use the site should in my opinion be more important than having a nice animation for sighted people. 

100% agree. I forgot about that aspect while diving into the reason of the problem.

> How about just using jQuery for that one event listener (with a descriptive comment)? We'll have to go over this again anyway once we reactify it and then we can decide wholesale whether we want to drop bootstrap's JS components or use them via something like react-bootstrap (which reimplements bootstrap's JS without jquery).

I would prefer that approach compared to removing the `data-toggle` because this would add a whole lot of extra javascript.